libs/interfaces/evaluation/evaluation.interface.ts

```diff
@@ -2,9 +2,9 @@ import {IEvaluationTag} from '..';
 
 export interface IEvaluation {
   id: string;
-  readonly filename: string;
+  filename: string;
   readonly data: Record<string, any> | undefined;
-  readonly evaluationTags: IEvaluationTag[] | null;
+  evaluationTags: IEvaluationTag[] | null;
```

Comment:
This is most likely one of the many places where the null value should be removed.